gradle-plugins/compose/src/test/test-projects/misc/testResources/src/commonTest/kotlin/CommonUiTest.kt

```diff
@@ -0,0 +1,39 @@
+import androidx.compose.ui.test.ExperimentalTestApi
+import androidx.compose.ui.test.runComposeUiTest
+import app.group.resources_test.generated.resources.Res
+import app.group.resources_test.generated.resources.app_name
+import app.group.resources_test.generated.resources.test_string
+import kotlinx.coroutines.test.runTest
+import org.jetbrains.compose.resources.ExperimentalResourceApi
+import org.jetbrains.compose.resources.stringResource
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotEquals
+
+@OptIn(ExperimentalTestApi::class, ExperimentalResourceApi::class)
+class CommonUiTest {
+
+    @Test
+    fun checkTestResources() {
+        runComposeUiTest {
+            setContent {
+                val mainStr = stringResource(Res.string.app_name)
+                val testStr = stringResource(Res.string.test_string)
+                assertEquals("Compose Resources App", mainStr)
+                assertEquals("Common test", testStr)
+                assertNotEquals("Compose Resources App", testStr)
+            }
+        }
+    }
+
+    @Test
+    fun checkTestFileResource() {
+        runTest {
```

Comment:
If this test would be on web eventually, `runTest` result should be returned from the function:

```kotlin
@Test
fun fooTest() = runTest { ... }
```

In the future (after 1.8), this requirement might affect `runComposeUiTest` too.